src/components/TweetView/Tweet.scss

```diff
@@ -0,0 +1,36 @@
+.block {
+  border-bottom: 1px solid #e1e8ed;
+  border-left: 1px solid #e1e8ed;
+  border-right: 1px solid #e1e8ed;
+  padding: 10px 15px;
+  list-style: none;
+  background-color: white;
+  &:hover {
+    background-color: #f5f8fa;
```

Comment:
It'd be good to move all colour swatches into a separate _variables_ sass file to introduce some consistency with the palette.